packages/next/client/image.tsx

```diff
@@ -723,29 +784,79 @@ export default function Image({
           ) : null}
         </span>
       ) : null}
+      <ImageElement raw={false} {...imgElementArgs} />
+      {priority ? (
+        // Note how we omit the `href` attribute, as it would only be relevant
+        // for browsers that do not support `imagesrcset`, and in those cases
+        // it would likely cause the incorrect image to be preloaded.
+        //
+        // https://html.spec.whatwg.org/multipage/semantics.html#attr-link-imagesrcset
+        <Head>
+          <link
+            key={
+              '__nimg-' +
+              imgAttributes.src +
+              imgAttributes.srcSet +
+              imgAttributes.sizes
+            }
+            rel="preload"
+            as="image"
+            href={imgAttributes.srcSet ? undefined : imgAttributes.src}
+            {...linkProps}
+          />
+        </Head>
+      ) : null}
+    </span>
+  )
+}
+
+const ImageElement = ({
+  raw,
+  imgAttributes,
+  heightInt,
+  widthInt,
+  qualityInt,
+  layout,
+  className,
+  imgStyle,
+  blurStyle,
+  lazy,
+  imgRef,
+  loading,
+  sizes,
+  srcString,
+  config,
+  unoptimized,
+  loader,
+  ...rest
+}: ImageElementProps) => {
+  return (
+    <>
       <img
         {...rest}
         {...imgAttributes}
+        {...(raw ? { height: heightInt, width: widthInt } : {})}
```

Comment:
Passing a `height` and a `width` has nothing to do with not applying NextJs responsive layout styles.

You're basically removing the possibility to add `height` and `width` on the image when using `raw` layout. Which is not what we need or want.

As users, we need to be able to pass `height` and `width`. We might not want to use NextJs styling, but we want the loading optimizations from the browsers. There's no need to remove this possibility from users.